c++/include/orc/Vector.hh

```diff
@@ -113,6 +115,36 @@ namespace orc {
     DataBuffer<int64_t> length;
   };
 
+  struct StringDictionary {
+    StringDictionary(MemoryPool& pool);
+    DataBuffer<char> dictionaryBlob;
+
+    // Offset for each dictionary key entry.
+    DataBuffer<int64_t> dictionaryOffset;
+
+    void getValueByIndex(int64_t index, char*& valPtr, int64_t& length)
+    {
```

Comment:
move the curly brace to the the previous line to follow the coding style. same applies to code below